tests/unit/test_routes/test_resource_update.py

```diff
@@ -282,3 +282,58 @@ def test_update_votes_authorization_header(
     assert (response.status_code == 200)
     assert (response.json['resource'].get(f"{UPVOTE}s") == initial_upvotes)
     assert (response.json['resource'].get(f"{DOWNVOTE}s") == initial_downvotes + 1)
+
+
+def test_delete_unused_languages(module_client, module_db,
```

Comment:
Can we get a test around deleting unused category as well? Would look just like this but be `test_delete_unused_category` and then of course it would test that the unused category was deleted.

After that, this is ready to merge!